requirements.txt

```diff
@@ -72,3 +72,4 @@ webencodings==0.5.1       # via bleach
 werkzeug==0.14.1          # via flask
 wtforms-json==0.3.3
 wtforms==2.2.1            # via flask-wtf, wtforms-json
+requests-kerberos==0.12.0
```

Comment:
This should be in extras

Reply:
+1 let's create a kerberos group in extra_requires